tests/src/Kernel/fixtures/markup_rendering_patterns/featured_media.yml

```diff
@@ -263,3 +263,77 @@ featured_media_iframe_with_text_container_right_aligned_with_link_ratio_21x9:
     contains:
       'div.col-12.col-md-6.order-md-2': 'Lorem Ipsum dolor sit amet.'
       'div.col-12.col-md-6.order-md-2 a': 'View all'
+featured_media_with_subtitle_and_subtitle_tag:
+  render:
+    '#type': pattern
+    '#id': featured_media
+    '#fields':
+      title: 'Main Title'
+      subtitle: 'Subtitle'
+      subtitle_tag: 'h6'
+  assertions:
+    count:
+      'h6.text-secondary': 1
+      'h2.bcl-heading': 1
+featured_media_with_subtitle_and_no_subtitle_tag_and_title:
+  render:
+    '#type': pattern
+    '#id': featured_media
+    '#fields':
+      title: 'Main Title'
+      subtitle: 'Subtitle'
+  assertions:
+    count:
+      'h3.text-secondary': 1
+      'h2.bcl-heading': 1
+featured_media_with_subtitle_empty_title_and_subtitle_tag:
+  render:
+    '#type': pattern
+    '#id': featured_media
+    '#fields':
+      title: ''
+      subtitle: 'Subtitle Here'
+  assertions:
+    count:
+      'h2.text-secondary': 1
+      'h2.bcl-heading': 0
+featured_media_with_bc_on_subtitle_and_subtitle_tag:
+  render:
+    '#type': pattern
+    '#id': featured_media
+    '#fields':
+      title: 'Main Title'
+      subtitle: 'Subtitle'
+      subtitle_tag: 'h6'
+  assertions:
+    count:
+      'h5.text-secondary': 1
```

Comment:
it has bc on setting

Reply:
i think the description is wrong... the implementation is correct as it is.... the bc is to use it as the previous version was (with h5).. if i set it as you said if on a project they already set a subtitle tag and enable the bc, the tag would still be in there and not a h5 as it should

Reply:
The BC setting provides only the default. Since the subtitle_tag parameter was not there before, the BC setting is used to determine its default status.
If a user purposely passes any value to it, that's the value to be used, not the h5.